features/environment.py

```diff
@@ -126,5 +127,8 @@ def get_or_create_instance_when_set(spf_id):
 
             ValidationOutcome.objects.bulk_create(outcomes_to_save)
 
-    else: # invoked via console
-        pass
+    else: # invoked via console or CI/CD pipeline
+        outcomes = [outcome.to_dict() for outcome in context.gherkin_outcomes]
+        outcomes_json_bytes = orjson.dumps(outcomes) #encoded to utf-8 
```

Comment:
In case you're looking for `b""` byte strings and that's the reason you picked `orjson`. You can also do:

```suggestion
        outcomes_json_bytes = json.dumps({}).encode('utf-8')
```

Reply:
That was indeed the reason and how it's resolved :)